pkg/bootstrapper/bootstrapper.go

```diff
@@ -569,6 +575,10 @@ func (wmcb *winNodeBootstrapper) generateInitialKubeletArgs(args map[string]stri
 	if wmcb.nodeIP != "" {
 		kubeletArgs = append(kubeletArgs, "--node-ip="+wmcb.nodeIP)
 	}
+	if !wmcb.dockerRuntime {
+		kubeletArgs = append(kubeletArgs, "--container-runtime=remote",
```

Comment:
It was suggested by @sebsoto in this [comment](https://github.com/openshift/windows-machine-config-bootstrapper/pull/322#discussion_r816848050), so it can remain as a reference within code.